README.md

```diff
@@ -292,6 +292,16 @@ govuk-docker compose run mysql mysql -h mysql -u root --password=root -e "CREATE
 pv whitehall_production.dump.gz | gunzip | govuk-docker compose run mysql mysql -h mysql -u root --password=root whitehall_development
 ```
 
+*For Whitehall*
+
+In order to ensure that Whitehall runs smoothly, your Docker settings may need to be changed in order to handle Whitehall. By default, Docker defaults to using 2 CPU cores and 2GB of RAM, and this is simply not enough to render Whitehall in a Docker container.
+
+You should set it to use at least 4 CPU cores and 8GB of RAM - 6 CPUs and 8 GB of RAM seems to work ok. Open the Docker dropdown via the whale icon in the OSX menu bar and selecting the `Advanced` option.
+
+You should also increase the Docker disk image size to the maximum.
```

Comment:
Do you know what level people are using to hit the max? I've not changed mine from the default of 64 GB and have been fine to import Whitehall (my MySQL usage is 22GB.

As the maximum might well be relative it might be best to specify a number - 100GB is probably plenty for most usages, whereas if you plan to clone the whole of GOV.UK you may need more still.

Reply:
I really don't have 100 GB and it seemed I had less than the default. Amended for that case in https://github.com/alphagov/govuk-docker/pull/182/commits/c9a83d33effc7f0402cdeb636eb9f2a512f402b9

Reply:
I really don't have 100 GB and it seemed I had less than the default. Amended for that case in https://github.com/alphagov/govuk-docker/pull/182/commits/c9a83d33effc7f0402cdeb636eb9f2a512f402b9